account_move_line_tax_editable/models/account_move_line.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountMoveLine(models.Model):
+
+    _inherit = 'account.move.line'
+
+    is_tax_editable = fields.Boolean(
+        string="Is tax data editable?", compute='_compute_is_tax_editable')
+
+    @api.multi
+    @api.depends('move_id.state')
```

Comment:
@benwillig @benwillig To the best of my knowledge, @api.depends is important for non-stored fields too.